edge-hub/core/test/Microsoft.Azure.Devices.Routing.Core.Test/MessageQueueIdHelperTest.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Routing.Core.Test
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
+    using Xunit;
+
+    public class MessageQueueIdHelperTest
+    {
+        [Fact]
+        [Unit]
+        public void TestGetMessageQueueIdWithDefaultPriority()
+        {
+            // Message queue with default priority should remain as endpoint id
+            var endpointId = Guid.NewGuid().ToString();
+            var messageQueueId = MessageQueueIdHelper.GetMessageQueueId(endpointId, Core.RouteFactory.DefaultPriority);
+            Assert.Equal(endpointId, messageQueueId);
+        }
+
+        [Fact]
+        [Unit]
+        public void TestGetMessageQueueIdWithNonDefaultPriority()
+        {
+            // Message queue with non-default priority should combain endpoint id and priority with MessageQueueIdHelper.MessageQueueIdDelimiter
+            var endpointId = Guid.NewGuid().ToString();
+            uint priority = 1234;
+            var expectMessageQueueId = $"{endpointId}{MessageQueueIdHelper.MessageQueueIdDelimiter}{priority}";
```

Comment:
```suggestion
            var expectMessageQueueId = $"test_Pri1234";
```
I encourage to use actual strings instead of referencing consts from prod code and string format. https://youtu.be/dJUVNFxrK_4?t=478

Reply:
It sounds reasonable but it's the same. The fact of where actual strings come from is from human calculation which based on coder's understanding of the logic, which is `$"{endpointId}{MessageQueueIdHelper.MessageQueueIdDelimiter}{priority}"` here. I won't say it;s the same code since I'm not asserting `func(a,b)` equals `func(a,b)`. If someone modified original function, the test will catch it. I couldn't agree his opinion UT is for catching bugs in code. IMPO, UT is used for avoid regression. And since `MessageQueueIdDelimiter` is defined as `const`, it could be changed as well. I see benefit hardcode in UT but for long term, it could be issue for maintains.